src/sentry/runner/settings.py

```diff
@@ -157,6 +157,7 @@
 SENTRY_WEB_PORT = 9000
 SENTRY_WEB_OPTIONS = {
     # 'workers': 3,  # the number of web workers
+    # 'protocol': 'uwsgi',  # Enable uwsgi protocol instead of http
```

Comment:
I'm intentionally adding this here for the `sentry.conf.py` template that spits out on `sentry init`.